File: Examples/CSharp/AsposePDF/DocumentConversion/XPSToPDF.cs
//////////////////////////////////////////////////////////////////////////
// Copyright 2001-2013 Aspose Pty Ltd. All Rights Reserved.
//
// This file is part of Aspose.Pdf. The source code in this file
// is only intended as a supplement to the documentation, and is provided
// "as is", without warranty of any kind, either expressed or implied.
//////////////////////////////////////////////////////////////////////////
using System.IO;

using Aspose.Pdf;

namespace CSharp.AsposePdf.DocumentConversion
{
    public class XPSToPDF
    {
        public static void Run()
        {
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_AsposePdf_DocumentConversion();

            // instantiate LoadOption object using XPS load option
            Aspose.Pdf.LoadOptions options = new XpsLoadOptions();
            
            // create document object 
            Aspose.Pdf.Document document = new Aspose.Pdf.Document(dataDir + "test.xps", options);
            
            // save the resultant PDF document
            document.Save(dataDir + "resultant.pdf");
        }
    }
}

File: Examples/VisualBasic/AsposePDF/DocumentConversion/XPSToPDF.vb
Imports System.IO
Imports System
Imports Microsoft.VisualBasic
Imports Aspose.Pdf

Namespace AsposePDF.DocumentConversion
    Public Class XPSToPDF
        Public Shared Sub Run()
            Try
                ' ExStart:XPSToPDF
                ' The path to the documents directory.
                Dim dataDir As String = RunExamples.GetDataDir_AsposePdf_DocumentConversion()

                ' Instantiate LoadOption object using XPS load option
                Dim options As LoadOptions = New XpsLoadOptions()

                ' Create document object 
                Dim document As New Document(dataDir & Convert.ToString("XPSToPDF.xps"), options)

                ' Save the resultant PDF document
                ' ExEnd:XPSToPDF
                document.Save(dataDir & Convert.ToString("XPSToPDF_out_.pdf"))
            Catch ex As Exception

                Console.WriteLine(ex.Message)
            End Try
        End Sub
    End Class
End Namespace